transaction-ballerina/build.gradle

```diff
@@ -152,15 +152,16 @@ task ballerinaTest {
     }
 
     def distributionBinPath =  project.projectDir.absolutePath + "/build/target/extracted-distributions/jballerina-tools-zip/jballerina-tools-${ballerinaLangVersion}/bin"
-
+    def testParams = "--code-coverage --includes=ballerinai.*:org.ballerinalang.stdlib.transaction.*"
```

Comment:
We can move the testParams out of the task and re-use it in other tasks as well. WDYT?